roles/reproducer/tasks/main.yml

```diff
@@ -104,6 +104,15 @@
         - bootstrap_env
         - bootstrap_layout
 
+- name: Bootstrap nat64 if needed
+  when:
+    - cifmw_use_libvirt | default(false) | bool
+    - cifmw_use_nat64 | default(false) | bool
+  tags:
+    - bootstrap_nat64
+  ansible.builtin.include_tasks:
+    file: nat64_appliance.yml
```

Comment:
When using `include_tasks` you want to also `apply` tags:

```
ansible.builtin.include_tasks:
  file: nat64_appliance.yml
  apply:
    tags:
      - bootstrap_nat64
      - bootstrap_layout
      - bootstrap_env
```
Else it won't do any task if you use `--tags bootstrap_nat64`

Reply:
Done